mypy/build.py

```diff
@@ -3488,7 +3488,7 @@ def process_stale_scc(graph: Graph, scc: list[str], manager: BuildManager) -> No
 
 
 def sorted_components(
-    graph: Graph, vertices: AbstractSet[str] | None = None, pri_max: int = PRI_ALL
+    graph: Graph, vertices: AbstractSet[str] | None = None, pri_max: int = PRI_INDIRECT
```

Comment:
The name `pri_max` is a bit confusing -- dependencies with a priority less than this are considered. I don't want to change this logic at this point, since it would require larger changes as this function is used in a few different places.